### PR TITLE
Add an introductory message in the top-level README.md

### PR DESCRIPTION
**Pull request by sunfishcode**
And some basic top-level text, and a few trivial corrections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+for WebAssembly.
+
+At this time, the contents of this repository are under development and known
+to be incomplet and inkorrect.
+
+Participation is welcome. Discussions about new features, significant semantic
+changes, or any specification change likely to generate substantial discussion
+should take place
+[the WebAssembly design repository](https://github.com/WebAssembly/design)
+first, so that this spec repository can remain focused.
```
Comment (jfbastien) [inline README.md]: When contributing, please follow our [Code of Ethics and Professional Conduct](https://github.com/WebAssembly/design/blob/master/CodeOfConduct.md).

**Comment by jfbastien** [comment]
Can you also add [Contributing.md](https://github.com/WebAssembly/design/blob/master/Contributing.md) and a link to it? github auto-recognizes it and puts a link at the top when creating issues / PRs.

**Comment by sunfishcode** [comment]
@jfbastien Done. I kept it minimal to avoid duplication.

**Comment by jfbastien** [comment]
lgtm

**Pull request merged**